lang/java/reef-io/src/main/java/org/apache/reef/io/network/ConnectionFactory.java

```diff
@@ -34,4 +34,14 @@
    * @return a connection
    */
   Connection<T> newConnection(Identifier destId);
+
+  /**
+   * @return the connection factory identifier.
+   */
+  Identifier getConnectionFactoryId();
+
+  /**
+   * @return the current node's end point id for the connection.
```

Comment:
`ConnectionFactory` creates multiple `Connection`s. Is this the end point id for the connection factory?